src/main/resources/buildServerResources/editOAuthSchemeProperties.jsp

```diff
@@ -5,6 +5,11 @@
     <jsp:include page="/admin/allowCreatingNewUsersByLogin.jsp"/>
 </div>
 <br/>
+<style type="text/css">
+    div.posRel {
+        width: 95%;
```

Comment:
why do we need this change?

Reply:
Sorry, I don't exactly remember. Will see If I can find a running TC instance to try with and without this change. Please try yourself if you have access to one